tree/treeplayer/test/dataframe/vecops_tvecallocator.cxx

```diff
@@ -0,0 +1,77 @@
+#include <ROOT/TVec.hxx>
+
+#include <gtest/gtest.h>
+
+#include <vector>
+#include <iostream>
+using namespace ROOT::Detail::VecOps;
+using namespace ROOT::Experimental::VecOps;
+
+TEST(TVecAllocator, ReusePointer)
+{
+   std::vector<double> vreference{1, 2, 3,4,5,6,7,8,9,10};
+
+   std::vector<double> vmodel (vreference);
+   TVecAllocator<double> alloc0(vmodel.data(), vmodel.size());
+
+   EXPECT_EQ(vmodel.data(), alloc0.allocate(123));
+
+   TVecAllocator<double> alloc1(vmodel.data(), vmodel.size());
+
+   std::vector<double, TVecAllocator<double>> v(alloc1);
+   v.resize(vmodel.size());
+
+   EXPECT_EQ(vmodel.data(), v.data());
+   EXPECT_EQ(vmodel.size(), v.size());
+
+   for (size_t i=0; i < vmodel.size(); ++i) {
+      EXPECT_EQ(vmodel[i],vreference[i]);
+      EXPECT_EQ(v[i],vreference[i]);
+   }
+
+   v.emplace_back(3.);
+   EXPECT_NE(vmodel.data(), v.data());
+
+   auto res = 0;
+   try {
+      v.resize(-1);
```

Comment:
Please use a standard thing like `std::numeric_limits<size_t>::max()` or `INT64_MAX` or, if you must, use `~0UL` instead of `-1` (note that `-1` may evaluate to `~0U == 4294967295`, not `~0UL == 18446744073709551615`, which is likely what you want here).

Reply:
I agree, thanks.

Reply:
I agree with using something more explicit, but the code as is is not ambiguous, you will get a vector with size `std::size_t(-1)` which is well defined